.github/workflows/build.yml

```diff
@@ -176,7 +176,7 @@ jobs:
         node-version: ${{ env.NODE_VERSION }}
     - uses: actions/checkout@v4
     - name: Download docker images artifacts
-      uses: actions/download-artifact@v3
+      uses: actions/download-artifact@v4
```

Comment:
If this version is out of sync with the "upload-artifact" version then the command fails quietly.